src/platform/Darwin/BlePlatformDelegate.h

```diff
@@ -30,14 +30,16 @@ namespace Internal {
 class BlePlatformDelegateImpl : public Ble::BlePlatformDelegate
 {
 public:
-    virtual bool SubscribeCharacteristic(BLE_CONNECTION_OBJECT connObj, const ChipBleUUID * svcId, const ChipBleUUID * charId);
-    virtual bool UnsubscribeCharacteristic(BLE_CONNECTION_OBJECT connObj, const ChipBleUUID * svcId, const ChipBleUUID * charId);
-    virtual bool CloseConnection(BLE_CONNECTION_OBJECT connObj);
+    virtual CHIP_ERROR SubscribeCharacteristic(BLE_CONNECTION_OBJECT connObj, const ChipBleUUID * svcId,
+                                               const ChipBleUUID * charId);
+    virtual CHIP_ERROR UnsubscribeCharacteristic(BLE_CONNECTION_OBJECT connObj, const ChipBleUUID * svcId,
```

Comment:
If these are overrides, should they be marked as "override", not "virtual"?

Maybe a followup PR, or just let me know when this lands and I can fix it at least for Darwin.

Reply:
Yes, that should be fixed. I'll try to do it in this PR so we will not forget about that.